Source/Execution/ExecutionContextConfigurator.cs

```diff
@@ -5,12 +5,14 @@
 
 using Dolittle.Applications;
 using Dolittle.Execution;
+using Dolittle.Lifecycle;
 using Dolittle.Security;
 using Dolittle.Tenancy;
 
 namespace Dolittle.AspNetCore.Execution
 {
     /// <inheritdoc/>
```

Comment:
Just noticed this; implementations typically don't inherit docs for the class description as the interface "defines" the contract and the class "represents" an implementation of it.